src/openms/source/KERNEL/MSSpectrum.cpp

```diff
@@ -166,10 +166,11 @@ namespace OpenMS
   void MSSpectrum::clear(bool clear_meta_data)
   {
     ContainerType::clear();
-    ContainerType::shrink_to_fit(); 
```

Comment:
@jpfeuffer this comes from the proteomicLFQ branch de7d4d019551dc3f5972d1c353ddd5fbd and at least here (for benchmark workloads) it does make a substantial difference of about 50% in speed due to re-allocation of memory. In general, I think its confusing if `clear()` does other things in OpenMS than what one is used from the STL.

Reply:
I think we can change that back and just expose shrink to fit externally.